Manual/Tutorial.tex

```diff
@@ -110,10 +110,11 @@ \subsection{Implementation}
 objects of the same category.
 
 After the declarations of the constructors we create a global category \GAP object, which will be the category $\Svec$.
+The call of \texttt{CreateCapCategory} lets \CapPkg create a category with name
+``\texttt{Skeletal category of rational vector spaces}'' such that every \GAP object which is an object of $\Svec$, lies in the filter \texttt{IsSQVecObj}
+and every \GAP object which is a morphism of $\Svec$, lies in the filter \texttt{IsSQVecMor}.
```

Comment:
Please also mention the role of `IsSQVecCat` here and remove the commas before `lies`.

Reply:
Done.

Reply:
Sorry, I missed another thing which I think should be mentioned here: the role of `IsCapCategoryTwoCell`. Or maybe we simply make the two cell filter optional? The only place I know of where this is used is `CapCat`.

Reply:
Everywhere I changed things to `CreateCapCategory` I also only wrote `IsCapCategoryTwoCell`. So maybe yes? :)

Reply:
I thought about this and I'm a bit reluctant about adding this convenience: If we add such a convenience for `CreateCapCategory`, we might also want to add it for `CreateCapCategoryWithDataTypes`. But `CreateCapCategoryWithDataTypes` is a function, not an operation, so we would have to do method selection via the number of arguments. This all sounds like it might cause problems if we want to add other kinds of convenience later on. Hence I suggest to keep things as they are and simply mention the role of `IsCapCategoryTwoCell`, i.e. something like "We do not want to consider two cells in the category, so we simply pass the generic filter."